src/templates/landing_pages/usb.jsx

```diff
@@ -0,0 +1,334 @@
+import React from 'react';
+import Layout from './_common/layout.jsx';
+
+const UsbPage = () => (
+    <Layout
+        meta_description={`${it.broker_name} Introducing USB`}
+        css_files={[
+            it.url_for('css/usb_style.css'),
+            'https://style.binary.com/binary.css',
+            'https://style.binary.com/binary.more.css',
+        ]}
+        js_files={[
+            'https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js',
+            'https://style.binary.com/binary.more.js',
+            'https://academy.binary.com/js/animate.min.js',
+            it.url_for('js/landing_pages/common.js'),
+            it.url_for('js/landing_pages/usb_page.js'),
+        ]}
+    >
+
+        <div id='page-top' />
+        <nav className='navbar navbar-custom navbar-fixed-top' role='navigation'>
+            <div className='navbar-container'>
+                <div className='nav-col-left'>
+                    <div className='logo-wrapper'>
+                        <a className='logo-parent logo-scroll' href='#page-top'>
+                            <div className='logo'>
+                                <div />
+                            </div>
+                            <div className='mobile-hide binary-logo-text'>
+                                <div />
+                            </div>
+                        </a>
+                    </div>
+                </div>
+                <div className='nav-col-right'>
+                    <div id='push' className='grill grill-ico' />
+                    <div id='push-nav' className='topnav'>
+                        <a className='page-scroll' href='#video'>{it.L('Overview')}</a>
```

Comment:
please remove all these `it.L`'s again since we don't want to translate this page 😞